src/main/java/gov/cdc/dataingestion/commands/Hl7Validation.java

```diff
@@ -0,0 +1,65 @@
+package gov.cdc.dataingestion.commands;
+
+import gov.cdc.dataingestion.model.AuthModel;
+import gov.cdc.dataingestion.util.AuthUtil;
+import gov.cdc.dataingestion.util.PropUtil;
+import picocli.CommandLine;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Properties;
+
+@CommandLine.Command(name = "validation", mixinStandardHelpOptions = true, description = "This functionality will validate the provided HL7 message.")
```

Comment:
we are validating HL7 here, so `validatehl7` as name for the command would suit better I guess. but it's upto you